plugins/mongodb/examples/mongodb-test.ecl

```diff
@@ -57,6 +57,13 @@ layoutDates := {STRING bucket_start_date, STRING bucket_end_date};
 layoutEmployee := {INTEGER1 id, STRING25 first, STRING25 last, REAL salary};
 layoutperson := {String username, String address, String email};
 
+layoutRegexTimestamp := RECORD
```

Comment:
Is this definition (and the associated test code) conflating testing regex and timestamp data types?  I feel like the new data types should be defined and tested separately, for clarity.

Reply:
Separated the test cases.